README.md

```diff
@@ -1,6 +1,10 @@
 # Super Zombie Fortress [![Action Status](https://github.com/redsunservers/SuperZombieFortress/workflows/Package/badge.svg)](https://github.com/redsunservers/SuperZombieFortress/actions?query=branch%3Amaster)
 
-Custom Team Fortress 2 Gamemode, inspired from Left 4 Dead
+Custom Team Fortress 2 Gamemode, inspired by Left 4 Dead.
+
+There's 2 teams, Survivors (RED) and Zombies (BLU). The Survivors start with melee weapons and must survive for x amount of time while collecting weapons/ammo that are found around the map.
```

Comment:
"survive for x amount of time" is only for survival maps which is not that popular, while more known maps require survivors to capture all control points or to reach the end of the map. Really it's entirely up to the map to determine what's the goal for survivors.

Reply:
Ah okay, I don't know tooo much about this plugin, is there any other objectives that I should know about?

Reply:
Some maps have a pickup system as an objective, and some has boss fights. But both is really for more map specific when it can be seen the same as "reach to the end of the map".

Reply:
So it's kind of like "reach the end of the map" while having to go through 'events' like a boss fight or picking up things along the way

Reply:
I'll update the pull request later, LMK if there's more inconsistencies or things I should add in that case